doc/source/train/user-guides/checkpoints.rst

```diff
@@ -174,6 +174,103 @@ appropriately in distributed training.
                         # Report to train session
                         checkpoint = Checkpoint.from_directory(tmpdir)
                         ray.train.report(metrics=metrics, checkpoint=checkpoint)
+
+
+    .. tab-item:: HuggingFace Transformers
+
+        Ray Train leverages HuggingFace Transformers Trainer's Callback interface 
+        to report metrics and checkpoints. 
+        
+        **Option 1: Use Ray Train's default report callback**
+        
+        We provide a simple callback implementation :class:`~ray.train.huggingface.RayTrainReportCallback` that 
+        reports on checkpoint save. It collects the latest reported logs and report it together with the 
+        latest saved checkpoint.
+
+        .. code-block:: python
+            :emphasize-lines: 11
+
+            from ray.train.huggingface import (
+                RayTrainReportCallback,
+                prepare_trainer
+            )
+            from ray.train.torch import TorchTrainer
+
+            def train_func(config):
+                ...
+
+                trainer = transformers.Trainer(...)
+
+                trainer.add_callback(RayTrainReportCallback())
+                
+                trainer = prepare_trainer(trainer)
+                trainer.train()
+            
+            ray_trainer = TorchTrainer(
+                train_func,
+                run_config=RunConfig(
+                    checkpoint_config=CheckpointConfig(
+                        num_to_keep=3,
+                        checkpoint_score_attribute="eval_loss",
+                        checkpoint_score_order="min",
+                    )
+                )
+            )
+        
+        You should properly configure the `logging_strategy`, `save_strategy` 
+        and `evaluation_strategy`, so that at the checkpoint saving step, transformers 
+        trainer also reports the latest monitoring metrics (e.g. `eval_loss` in the above case).
```

Comment:
I think it would be helpful to include these directly in the example itself, as it's currently not being shown. 

In other words, the example should make it clear that our callback is complementary to Transformers logging/checkpointing.

Reply:
Make sense. Like explicitly define the `save_strategy` and `evaluation_strategy`, and put these info as comments in the example?